src/demo/types.ts

```diff
@@ -0,0 +1,9 @@
+export type StringifiedBoolean = 'true' | 'false'
+
+export type ApplicantData = {
+  first_name?: string
+  last_name?: string
+  email?: string
+}
+
+export type DecoupleResponseOptions = 'success' | 'error' | 'onfido'
```

Comment:
What's the `onfido` response?

Reply:
It seems to be no longer needed :thinking: Will remove it

Reply:
It was used here: https://github.com/onfido/onfido-sdk-ui/blob/f5cf702a33a6d6fe0aaa0d17d061ffcb250a2289/src/demo/demoUtils.ts#L246-L253. Seems to be a missing typings from the old features